LDAR_Sim/testing/unit_testing/test_scheduling/test_survey_planner/test_update_with_latest_survey.py

```diff
@@ -0,0 +1,253 @@
+"""
```

Comment:
Little nits here, you can decide whether or not address: You use the same names for all the detection records and dates so technically they overwrite. In general this is not the best practice. Also it would be nice to move making those into other methods so reading the test is easiest but it does seem like a bit of work.